### PR TITLE
[libvpx] Update to 1.13.1

### LINKED ISSUE #34809: [libvpx] update to 1.13.1
### Library name

libvpx

### New version number

1.13.1

### Other information that may be useful (release notes, etc...)

libxvpx prior to version 1.13.1 suffers from a buffer overflow vulnerability, see CVE-2023-5217.

### PR DESCRIPTION
**Pull request by LilyWangLL**
Fixes #34809 

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [ ] ~~The "supports" clause reflects platforms that may be fixed by this new version~~
- [ ] ~~Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~~
- [ ] ~~Any patches that are no longer applied are deleted from the port's directory.~~
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

All features passed with following triplets:
```
x86-windows
x64-windows
x64-windows-static
```
Usage test passed on `x64-windows`.

### CODE REVIEW

File: ports/libvpx/vpx.pc.in [hunk 1]
```diff
@@ -1,12 +1,12 @@
 prefix=@LIBVPX_PREFIX@
-# pkg-config file from libvpx v1.10.0
+# pkg-config file from libvpx v1.13.1
```
Comment (dg0yt) [inline ports/libvpx/vpx.pc.in:2]:
```suggestion
```
This comment has no value.

**Comment by LilyWangLL** [comment]
Just remove a comment, it no need to occupy ci resources again.

**Review by FrankXie05 — APPROVED** [review]

**Comment by basilgello** [comment]
libvpx perfectly builds and runs on arm-android-neon and arm64-android with the edits from my commit.

**Comment by basilgello** [comment]
@LilyWangLL I fixed arm/arm64 build for Android in my port for rustdesk (see above). Hope it saves you anothervround of double work :)

**Comment by basilgello** [comment]
@dg0yt Can you please look at my port in private space? I tailored it specifically for rustdesk but maybe you can take it to go ahead with this PR :)

**Comment by jlaine** [comment]
Any chance of getting this merged as it addresses a security issue?

**Comment by basilgello** [comment]
I think I will hijack the MR if @LilyWangLL is OK

**Comment by LilyWangLL** [comment]
Closed as duplicate of https://github.com/microsoft/vcpkg/pull/35047.

**Pull request closed without merging**